bench_runner/scripts/install.py

```diff
@@ -190,6 +195,9 @@ def generate_benchmark(dst: Any) -> Any:
         for flag in flags.FLAGS
     )
 
+    if get_skip_publish_mirror():
+        del dst["jobs"]["publish"]
```

Comment:
If this step is deleted, the `notify` step will also need to be updated to not depend on it.

```yaml
  notify:
    uses: ./.github/workflows/_notify.yml
    if: ${{ always() }}
    needs: [publish, determine_base]
```

So add a line here:

```
dst["jobs"]["notify"]["needs"] = ["generate", "determine_base"]
```